collect_app/src/androidTest/java/org/odk/collect/android/support/pages/ErrorPage.kt

```diff
@@ -8,12 +8,18 @@ import androidx.test.espresso.matcher.ViewMatchers.withParent
 import org.hamcrest.Matchers.allOf
 import org.hamcrest.Matchers.instanceOf
 import org.odk.collect.android.R
+import org.odk.collect.android.support.WaitFor.waitFor
+import java.util.concurrent.Callable
 
 class ErrorPage : Page<ErrorPage>() {
 
     override fun assertOnPage(): ErrorPage {
-        assertText(R.string.errors)
-        return this
+        return waitFor(
```

Comment:
Can we move these `waitFor` calls into the `NotificationDrawer`? I can totally understand us needing `waitFor` when returning from a notification, but we shouldn't need it when moving from one page within the app to the other.